src/test/java/net/masterthought/cucumber/ReportParserTest.java

```diff
@@ -88,4 +90,119 @@ public void parseJsonResults_OnNoExistingFile_ThrowsException() {
 
         reportParser.parseJsonFiles(jsonReports);
     }
+
+    @Test
+    public void parsePropertyFiles_EmptyProperties() {
+
+        // given
+        initWithProperties(EMPTY_PROPERTIES);
+        ReportParser reportParser = new ReportParser(configuration);
+
+        // when
+        reportParser.parsePropertiesFiles(propertyReports);
+
+        // then
+        assertThat(configuration.getClassifications()).hasSize(0);
+    }
+
+    @Test
+    public void parsePropertyFiles_Populate_One_File() {
+
+        // given
+        initWithProperties(SAMPLE_ONE_PROPERTIES);
+        ReportParser reportParser = new ReportParser(configuration);
+
+        // when
+        reportParser.parsePropertiesFiles(propertyReports);
+
+        // then
+        assertThat(configuration.getClassifications()).hasSize(5);
+    }
+
+    @Test
+    public void parsePropertyFiles_Populate_Two_Files() {
+
+        // given
+        initWithProperties(SAMPLE_ONE_PROPERTIES, SAMPLE_TWO_PROPERTIES);
+        ReportParser reportParser = new ReportParser(configuration);
+
+        // when
+        reportParser.parsePropertiesFiles(propertyReports);
+
+        // then
+        assertThat(configuration.getClassifications()).hasSize(8);
```

Comment:
same

Reply:
not sure what you mean here...can you expand a bit on what you mean?

Reply:
`isEmpty()` :)

Reply:
I'm checking for size eight(8) tho

Reply:
validates size but could also validate order for at least some elements